docs/introduction/Cross-chain-transfer-protocol.rst

```diff
@@ -15,9 +15,20 @@ We can declare a new sidechain by using the following RPC command:
 
 .. code:: Bash
 
-   sc_create withdrawalEpochLength "address" amount "verification key" "vrfPublickKey" "genSysConstant"
+   sc_create {"withdrawalEpochLength": withdrawalEpochLength, "toaddress": address, "amount": creation_amount, "wCertVk": vk, "constant": constant, 'customData': custom_data, 'wCeasedVk': cswVk, 'vFieldElementCertificateFieldConfig': feCfg,
+              'vBitVectorCertificateFieldConfig': bvCfg, 'forwardTransferScFee': ftScFee, 'mainchainBackwardTransferScFee': mbtrScFee, 'mainchainBackwardTransferRequestDataLength': mbtrRequestDataLength}
+
+Parameters to the command must passed in json format. The command must specify where the first forward transfer coins are sent("toaddress"), it's amount("amount"), as well as the epoch length("withdrawalEpochLength"). It is the epoch length that defines the frequency, in blocks, of the backward transfers' submissions (see the “backward transfers” paragraph below). The sc_create command also includes the cryptographic key to receive coins back from the sidechain("wCertVk"). The verification key guarantees that the received coins were processed according to a matching proving system. Beside these parameters sc_create have some optional ones:
```

Comment:
Parameters to the command must be passed in JSON format. The command must specify where the first forward transfer coins are sent("toaddress"), its amount("amount"), as well as the epoch length("withdrawalEpochLength"). It is the epoch length that defines the frequency, in blocks, of the backward transfers' submissions (see the “backward transfers” paragraph below). The sc_create command also includes the cryptographic key to receive coins back from the sidechain("wCertVk"). The verification key guarantees that the received coins were processed according to a matching proving system. Besides these parameters sc_create have some optional ones: